clients/instance/ibm-pi-clonevolumes.go

```diff
@@ -96,7 +96,7 @@ func (f *IBMPICloneVolumeClient) DeleteClone(id string) error {
 	return nil
 }
 
-// Initiate the start clone request
+// Initiate a Start Clone Request
```

Comment:
Does the Go language need capitalized words in the comments?

Reply:
I don't think so.
I think its a good idea to follow some type of naming convention for comments going forward.